src/main/java/io/supertokens/webserver/WebserverAPI.java

```diff
@@ -486,7 +490,12 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
                 Logging.info(main, tenantIdentifier,
                         "API called: " + req.getRequestURI() + ". Method: " + req.getMethod(), false);
             }
-            super.service(req, resp);
+            String method = req.getMethod();
+            if(method.equals("PATCH")){
```

Comment:
are we sure that it will always return all capital letters and not something like `patch` ever?

Reply:
yes. I exactly followed the implementation in the tomcat HttpServletRequest with the other - supported - methods. Also, [RFC 2616](https://www.ietf.org/rfc/rfc2616.txt) states, that Method names are case-sensitive (section 5.1.1)